### PR TITLE
Updating configurations

### PR DESCRIPTION
**Pull request by bemer**
*Issue #, if available:*

*Description of changes:* Changing configurations to add support to HTTPS requests that will be needed in the TLS blogpost.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/eks-getting-started/infrastructure/appmesh_templates/appmesh-yelb-ui.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
       app: yelb-ui
   listeners:
     - portMapping:
-        port: 4567
+        port: 80
```
Comment (bcelenza) [inline walkthroughs/eks-getting-started/infrastructure/appmesh_templates/appmesh-yelb-ui.yaml:13]: How does this affect TLS?

File: walkthroughs/eks-getting-started/infrastructure/yelb_initial_deployment.yaml [hunk 2]
```diff
@@ -88,7 +88,7 @@ spec:
     spec:
       containers:
         - name: yelb-ui
-          image: mreferre/yelb-ui:0.6
+          image: mreferre/yelb-ui:0.7
```
Comment (bcelenza) [inline walkthroughs/eks-getting-started/infrastructure/yelb_initial_deployment.yaml:91]: What's the difference between these two versions?

**Comment by bcelenza** [comment]
Overall it's not clear how these changes add TLS. I would expect additional configuration changes for Virtual Nodes and Gateways, both for TLS termination and origination (client policies).

Additionally, how has this been tested?

**Comment by bemer** [comment]
Hey Brian, we are working in a different blog post around TLS with App Mesh. These changes are needed so we can use the same implementation as the base for this next blog post.

File: walkthroughs/eks-getting-started/infrastructure/appmesh_templates/appmesh-yelb-ui.yaml (hunk 1 above)
Comment (bemer) [reply walkthroughs/eks-getting-started/infrastructure/appmesh_templates/appmesh-yelb-ui.yaml:13]: Container was exposing port 80 but the virtual node was being accessed under the port 4567. Had to fix this.

File: walkthroughs/eks-getting-started/infrastructure/yelb_initial_deployment.yaml (hunk 2 above)
Comment (bemer) [reply walkthroughs/eks-getting-started/infrastructure/yelb_initial_deployment.yaml:91]: It was a change so the application can support https. Before it, http was hardcoded so all of the api calls to get stats and votes were being done over http.

File: walkthroughs/eks-getting-started/infrastructure/appmesh_templates/appmesh-yelb-ui.yaml (hunk 1 above)
Comment (egagala) [reply walkthroughs/eks-getting-started/infrastructure/appmesh_templates/appmesh-yelb-ui.yaml:13]: Neither of changes add directly support for TLS. However I'm working on App Mesh TLS blog post which needs to have proper base setup (without TLS) of the sampe yelb app already integrated with App Mesh. The changes here are to fix base configuration and prepare it for further integration with TLS which will be separate code.

1. change of yelb-ui version from 0.6 to 0.7 adds https suppor for this component. Refer here: https://github.com/mreferre/yelb/commit/048d6db7218fac7023137de018551539baa8ec1b

2. change of yelb-ui virtual node listener port from 4567 to 80 is just fix which set correct port to expose yelb-ui

Both changes were tested by deployment of the yelb app with new settings.

File: walkthroughs/eks-getting-started/infrastructure/yelb_initial_deployment.yaml (hunk 2 above)
Comment (egagala) [reply walkthroughs/eks-getting-started/infrastructure/yelb_initial_deployment.yaml:91]: Please refer to comment above

**Pull request merged**